aten/src/ATen/native/vulkan/ops/Mm.cpp

```diff
@@ -163,6 +307,37 @@ Tensor mm(const Tensor& self_arg, const Tensor& mat2_arg) {
 
 #ifdef USE_VULKAN_API
 
+TORCH_LIBRARY(vulkan_lin, m) {
```

Comment:
Can we reuse the same library and namespace as for conv 'vulkan_prepack' ?

Reply:
This can be done, but just as a heads up I'll have to do all the registrations in a new file (like `RegisterOpContextClass.cpp` for XNNPACK.